builder/proxmox/common/step_start_vm_test.go

```diff
@@ -441,3 +442,59 @@ func TestStartVMWithForce(t *testing.T) {
 		})
 	}
 }
+
+func TestStartVM_AssertInitialQuemuConfig(t *testing.T) {
```

Comment:
Added this test to make sure the value is actually inverted when the config is assembled